packages/wdio-cucumber-framework/src/index.js

```diff
@@ -73,9 +73,14 @@ class CucumberAdapter {
         let result
 
         try {
+            // we need tp ensure that we don't run the registration code
```

Comment:
this change can potentially go away if we only ever have one `Runner` instance that we reuse across runs.